tests/models/booking_test.py

```diff
@@ -387,3 +388,37 @@ def test_booking_on_event_is_not_cancellable_if_begining_date_time_before_72_hou
 
         # Then
         assert is_cancellable == False
+
+
+class BookingCancellationDateTest:
+    @clean_database
+    def test_should_update_cancellation_date_when_booking_is_cancelled(self, app):
+        # Given
+        user = create_user()
+        booking = create_booking(user=user, amount=1)
+
+        # When
+        booking.isCancelled = True
+        repository.save(booking)
```

Comment:
```suggestion
        # When
        repository.save(booking)
        booking.isCancelled = True
        repository.save(booking)
```

pour avoir un booking update et pas juste créé

Reply:
Du coup, je veux que si je sauvegarde un booking annulé, il y ait une date d'annulation.
J'ai ajusté le nom du test